examples/next/docs/js/demo/src/index.js

```diff
@@ -5,6 +5,9 @@ import "pikaday/css/pikaday.css";
 import { data } from "./constants";
```

Comment:
Can I ask you to revert that file? AFAIK the "demo" from the examples project is currently reserved for LTR. Probably something will be changed but only after the official RTL support release.